### PR TITLE
UQS design changes and info screen (EXPOSUREAPP-10470)

### PR DESCRIPTION
**Pull request by SamuraiKek**
What's new:

- UQS subtitle changed and added an info icon.
- Info icon will take you to the new QR Code information screen.

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -920,7 +920,7 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <!-- YTXT: instruction text for QR code scanning  -->
     <string name="qr_code_scan_body_title">"Welche QR-Codes können Sie scannen?"</string>
     <!-- YTXT: instruction text for QR code scanning  -->
-    <string name="qr_code_scan_body_subtitle">"PCR-Test und Schnelltests, Testzertifikate, Impfzertifikate, Genesenenzertifikate und Check-ins."</string>
+    <string name="qr_code_scan_body_subtitle">"Tests, Zertifikate, Check-ins und Überprüfung  bei Ticketbuchungen"</string>
```
Comment (KathrinLuecke) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:923]: Please remove superfluos space after "Überprüfung"

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/info/UqsInfoFragment.kt [hunk 2]
```diff
@@ -1,8 +1,37 @@
...
+
+        enterTransition = MaterialSharedAxis(MaterialSharedAxis.Z, true)
+        returnTransition = MaterialSharedAxis(MaterialSharedAxis.Z, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        binding.apply {
+            toolbar.setNavigationOnClickListener { popBackStack() }
+            privacyInformation.setOnClickListener {
+                findNavController().navigate("coronawarnapp://contactdiary/dataPrivacy".toUri())
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/info/UqsInfoFragment.kt:30]: I think you can just use the screen destination Id , they are in the same graph , why do you need to use `uri`
```kotlin
findNavController().navigate(R.id.privacyscreen)`
```

File: Corona-Warn-App/src/main/res/layout/fragment_uqs_info.xml [hunk 3]
```diff
@@ -1,12 +1,211 @@
...
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/booking_provider_details"
+                    app:titleText="@string/qr_code_info_screen_privacy_policy" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+        </androidx.core.widget.NestedScrollView>
+
+    </androidx.coordinatorlayout.widget.CoordinatorLayout>
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/fragment_uqs_info.xml:211]: Dark mode has different icon color , see Test icon
Screenshot 2021-11-17 at 12 25 41

**Comment by mtwalli** [comment]
@SamuraiKek  you have a conflict

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/info/UqsInfoFragment.kt (hunk 2 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/info/UqsInfoFragment.kt:30]: You're right, didn't see that.

File: Corona-Warn-App/src/main/res/layout/fragment_uqs_info.xml (hunk 3 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/layout/fragment_uqs_info.xml:214]: Updated the colors.

**Review by AlexanderAlferov — APPROVED** [review]
LGTM 🚀

**Review by mtwalli — APPROVED** [review]

**Pull request merged**